### DISCUSSION TITLE
Fix install for OpenBSD

### DISCUSSION THREAD

**Discussion by maximvarentsov** [Chia-Network/chia-blockchain#7695] [General]
Hello, i have fix for installing chia on OpenBSD 6.9 amd64. But can't edit wiki page anymore.

https://github.com/Chia-Network/chia-blockchain/wiki/OpenBSD-Install#install-chia-on-openbsd

Anyone can update page? 

-Tested with Chia 1.1.4 on OpenBSD/amd64 6.8
+Tested with Chia 1.2.2 on OpenBSD/amd64 6.9

-doas pkg_add git python-3.8.6 rust cmake gmake gmpxx
+doas pkg_add git python-3.8.11 rust cmake gmake gmpxx

-git clone https://github.com/Chia-Network/clvm_rs.git --branch 0.1.7
+git clone https://github.com/Chia-Network/clvm_rs.git --branch 0.1.8

+git clone https://github.com/pyca/cryptography.git --branch 3.4.7

+# build cryptography is broken for LibreSSL > 3.3.2 patching required
+cd ../cryptography
+curl -s  https://raw.githubusercontent.com/vishwin/freebsd-ports/a53c328ac558fee901aae92a7e5f9d1d13b41de8/security/py-cryptography/files/patch-Fix-build-with-LibreSSL-3.3.2-5988 | patch -p0
+pip install .

-sed -i 's|"chiavdf==1.0.1"|"chiavdf==1.0.2.dev1"|' setup.py
+sed -i 's|"chiavdf==1.0.2"|"chiavdf==1.0.2.dev2"|' setup.py

**Comment by emlowe** [comment]
Thanks, we needed to lock down the wiki due to some malicious edits, however, your wiki edit privileges will be restored - thanks for maintaining this page
